insights/core/__init__.py

```diff
@@ -631,6 +616,21 @@ def __init__(self, context, extra_bad_lines=None):
         super(CommandParser, self).__init__(context)
 
 
+class ContainerParser(CommandParser):
```

Comment:
From the collected content example `API_OUTPUT_BAD` (see in test file) of this Spec,  it seems that a container File collection will be collected via command `cat`.  

Based on this, both Container Command/File can be applied with `CommandParser` . 

Hi @wushiqinlou  , could you help confirm if this `CommandParser` usage here make sense in Container spec collection view ?